tests/functional/src/main/java/org/mule/functional/junit4/ApplicationContextBuilder.java

```diff
@@ -8,11 +8,14 @@
 
 import static java.util.Collections.emptyMap;
 import static java.util.Collections.singleton;
+import static org.mockito.Mockito.mock;
+
 import static org.mule.functional.junit4.FunctionalTestCase.extensionManagerWithMuleExtModelBuilder;
 import static org.mule.runtime.core.api.config.bootstrap.ArtifactType.APP;
 import static org.mule.runtime.core.api.extension.MuleExtensionModelProvider.getExtensionModel;
 
 import org.mule.runtime.api.meta.model.ExtensionModel;
+import org.mule.runtime.api.metadata.ExpressionLanguageMetadataService;
```

Comment:
Reviewed the imports. These were preserved because I used them in other parts of the class after removing the DW dependency from the "functional" module